documentation/testing/unit-tests.md

```diff
@@ -162,14 +162,14 @@ class SomeScreenInteractorTest {
 We can also unit test `View` implementation.
 In this test we should move `View` to some particular state and verify it by using `Espresso`.
 After that we can invoke some actions on `View` and verify that view sends events or invokes callbacks as expected.
-It can be easily done with the help of `RibViewTestRule`.
+It can be easily done with the help of `RibsViewRule`.
 
 ```kotlin
 class SomeViewImplTest {
 
     @get:Rule
-    val rule = RibViewTestRule { factoryContext ->
-        SomeScreenViewImpl.Factory().invoke(/* dependency */ null).invoke(/* view factory context */ factoryContext)
```

Comment:
We still need to do `invoke(null).invoke(factoryContext)` like [here](https://github.com/badoo/RIBs/blob/master/sandbox/src/androidTest/java/com/badoo/ribs/sandbox/rib/menu/MenuViewTest.kt#L25).